static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-liveblog.js

```diff
@@ -0,0 +1,54 @@
+define([
+    'common/modules/commercial/contributions-utilities',
+    'lib/geolocation',
+    'lodash/utilities/template',
+    'raw-loader!common/views/acquisitions-epic-liveblog.html'
+], function (
+    contributionsUtilities,
+    geolocation,
+    template,
+    liveblogEpicTemplate
+) {
+
+    return contributionsUtilities.makeABTest({
+        id: 'AcquisitionsEpicLiveblog',
+        campaignId: 'epic_liveblog',
+
+        start: '2017-04-01',
+        expiry: '2018-04-01',
+
+        author: 'Joseph Smith',
+        description: 'This places the epic underneath liveblog blocks which the author has specified in Composer should have an epic against them',
+        successMeasure: 'Member acquisition and contributions',
+        idealOutcome: 'Our wonderful readers will support The Guardian in this time of need!',
+
+        audienceCriteria: 'All',
+        audience: 1,
+        audienceOffset: 0,
+
+        pageCheck: function(page) {
+            return page.contentType === 'LiveBlog';
+        },
+
+        variants: [
+            {
+                id: 'control',
+                maxViews: {
+                    days: 30,
```

Comment:
I would just ask amanda if we need to do this, or whether the rules should be different, as the old liveblog posts were seen by everyone regardless of their previous epic views